independent-publisher-connectors/Rarible/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3240 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v0.1",
+    "title": "Rarible",
+    "description": "Rarible Multichain Protocol is a blockchain-agnostic and decentralized tool to query, issue, and trade NFTs.",
+    "contact": {
+      "name": "Roy Paar",
+      "email": "roypaar@microsoft.com"
+    }
+  },
+  "host": "ethereum-api.rarible.org",
+  "basePath": "/v0.1",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/erc20/balances/{contract}/{owner}": {
+      "get": {
+        "description": "Return an ERC20 token balance",
+        "summary": "Return an ERC20 token balance",
+        "tags": [
+          "erc20-balance-controller"
+        ],
+        "operationId": "getErc20Balance",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "contract",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Address of the contract",
+            "x-ms-summary": "Address of the contract",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Address of the ERC20 token owner",
+            "x-ms-summary": "Address of the ERC20 token owner",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Erc20DecimalBalance"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/Erc20IndexerApiError"
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/Erc20IndexerApiError"
+            }
+          }
+        }
+      }
+    },
+    "/nft/ownerships/{ownershipId}": {
+      "get": {
+        "description": "Return NFTs by Ownership ID",
+        "summary": "getNftOwnershipById",
```

Comment:
@RoyPaarMS All the summary and description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid summary and description values. **This comment applies for all the summary values in the swagger.**

Reply:
Corrected